client/src/components/SurveyInsights.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div>
+    <a-space direction="vertical" :size="24" style="width: 100%">
+      <a-card 
+        v-for="question in survey?.questions" 
+        :key="question.id"
+        :title="question.description || `问题${question.id}`"
```

Comment:
```suggestion
        :title="`${index + 1}. ${question.description}`"
```
这个 title 的渲染应该和 SurveyAnswer 的样式一样.（你注意 v-for 加个 index 参数）